aead/src/lib.rs

```diff
@@ -52,6 +56,8 @@ use generic_array::{typenum::Unsigned, ArrayLength, GenericArray};
 
 #[cfg(feature = "alloc")]
 use alloc::vec::Vec;
+use bytes::BytesMut;
+use std::io::Bytes;
```

Comment:
These aren't properly feature gated, and really if you're adding `bytes` support, I think it should be able to support an `alloc`-only profile